token/program-2022/src/extension/fixed_owner/mod.rs

```diff
@@ -0,0 +1,19 @@
+use {
+    crate::extension::{Extension, ExtensionType},
+    bytemuck::{Pod, Zeroable},
+};
+
+/// Fixed Owner extension instructions
+pub mod instruction;
+
+/// Fixed Owner extension processor
+pub mod processor;
+
+/// Indicates that the Account owner authority cannot be changed
+#[repr(C)]
```

Comment:
Not something we have to worry about for now, but unit structs are improper C types since they have size 0 in C, and size 1 in C++: https://github.com/rust-lang/rust/issues/20660

Reply:
Neat.  `repr(transparent)` works just as well in this case, thanks